pelican/contents.py

```diff
@@ -547,9 +547,29 @@ def refresh_metadata_intersite_links(self) -> None:
                 self._summary = self.metadata["summary"]
 
 
+class SkipStub(Content):
+    # Stub class representing content that should not be processed in any way
```

Comment:
```suggestion
    """Stub class representing content that should not be processed in any way."""
```
It's a perfect docstring